src/dune_config/config.mli

```diff
@@ -37,6 +37,9 @@ val copy_file : [ `Portable | `Best ] t
     Note that environment variables take precedence over the values passed here
     for easy overriding. *)
 
+(** Execute some actions in background threads *)
```

Comment:
It would be nice to specify which actions are affected by this ("some" is a little too vague).

Reply:
Done